src/content/docs/errors-inbox/apm-tab.mdx

```diff
@@ -0,0 +1,103 @@
+
+<img
+  title="group errors tab"
+  alt="A depiction of the group errors tab"
+  src={apmGroupErrorsTab}
+/>
+
+## Errors list view  [#list-view]
+
+As you try to investigate why the **Send Message** button isn't working in your application, the first place to start is the **Errors list view**. Here you can see error rate charts at a glance and determine if there are any unexpected spikes, dips, or patterns with errors in general.
+
+The **Errors list view** allows you to:
+
+* Correlate any general patterns on the top 5 errors chart to alerts occurring during the same time period.
+* Dynamic grouping: The default grouping for error occurrences is based on error message, error class, and Transaction UI name. You have the option to change the grouping options by any attribute, up to five attributes at a time.
+* Filtering: Many New Relic customers instrument custom attributes and filtering on a specific custom attribute can be a really quick way to cut through the noise of all error occurrences. 
+
+## Errors details view [#details-view]
+
+Let's say that after you investigated your errors as a whole using the list view, you identified a specific group of hosts that is causing an error spike. You can now use the **details view** to better understand in-context details.
+
+The **Details view** contains: 
+
+* **Applied filters & group-by values:** You may receive a permalink of a specific occurrence. It's important to know the filters and group-by selections to understand the context of the occurrence. 
+* **Profiles:** The profiles tab helps you observe interesting trends across all error occurrences inside a specific error group. You can download this information as a CSV file for further analysis. When you click on an attribute bar, a table is rendered showing the breakdown of attributes and counts.
+
+<img
+  title="error profiles"
+  alt="A depiction of error profiles in the UI"
+  src={apmErrorProfiles}
+/>
+
+### Error occurrences: [#error-occurrences]
+
+The detailed view also allows you to view specific errors. In the top right, you can navigate between the first instance of the error, the last, and any instance in between.
+
+## Triage [#triage-card]
+
+The triage section links the specific error occurrence you're viewing to a system-created error group that has a unique fingerprint. Why does that matter? That unique fingerprint  enables you to triage an error group using a status update or assignment. The system-created error groups are the ones you find on the Triage tab. For more info on how they are generated, see How error groups work.
+
+<img
+  title="triage errors"
+  alt="A depiction of how to tirage errors in APM"
+  src={apmTriageErrors}
+/>
+
+## Select the time period for error data [#selected-time-period]
+
+Use the time picker to examine the details of error events over the past week, month, etc. 
```

Comment:
We don't like to use etc. We would rather word as 'over the past week, month, or other time range.

Reply:
done!